packages/web3-providers/src/Chainbase/apis/FungibleTokenAPI.ts

```diff
@@ -65,9 +65,9 @@ export class ChainbaseFungibleTokenAPI implements FungibleTokenAPI.Provider<Chai
     }
 
     async getFungibleTokenPrice(chainId: ChainId, address: string) {
-        if (isNativeTokenAddress(address) || !isValidAddress(address) || !isValidChainId(chainId)) return undefined
+        if (isNativeTokenAddress(address) || !isValidAddress(address)) return undefined
         const data = await fetchFromChainbase<FT_Price>(
-            urlcat('/v1/token/price', { chain_id: chainId, contract_address: address }),
+            urlcat('/v1/token/price', { chain_id: ChainId.Mainnet, contract_address: address }),
```

Comment:
solana is using coingeko, chainbase is evm only.

Reply:
I don't think it's correct. The address could be from other EVM chains that don't exist on Mainnet and fail the API.